### DISCUSSION TITLE
Failed to upload attachment

### DISCUSSION THREAD

**Discussion by johe123qwe** [dani-garcia/vaultwarden#4473] [Q&A]
I have modified USER_SEND_LIMIT and USER_ATTACHMENT_LIMIT and it still fails when I upload files over 500 megabytes. Please help me, thank you!
![Monosnap Send | Vaultwarden Web 2024-04-05 23-08-25](https://github.com/dani-garcia/vaultwarden/assets/22858214/714c7ad4-054a-46f0-ace8-fa2dbc87acc2)

**Comment by BlackDex** [comment]
That is a client set fixed limit, see: https://bitwarden.com/help/attachments/
You can't change the maximum file size per upload for the clients, only how much a client is allowed to store.

This is due to client limitations and encrypting the files which are large in size.
